src/plots/gl3d/layout/axis_attributes.js

```diff
@@ -68,6 +68,35 @@ module.exports = {
         description: 'Sets whether or not this axis is labeled'
     },
     color: axesAttrs.color,
+    categorymode: {
```

Comment:
@etpinard good point! I'll await all PR responses and do them in one to not go wild with commits.